### PR TITLE
docs(ideas): add job listings filter

### PR DESCRIPTION
**Pull request by Vinitvh**
Added a new idea in ideas.json

### CODE REVIEW

File: data/ideas.json [hunk 1]
```diff
@@ -184,5 +184,12 @@
         "description": "An app or website to generate a possible, safest and most acceptable roadmap for the query user seraches about like full stack developement, software developement etc. and also shows the details of each step to gain success",
         "level": "advanced",
         "tags": "roadmap, success, process"
-    }
+    },
+    {
+        "title": "Job Listings Filter",
+        "description": "A Job listing webapp with filter feature for different tags",
+        "level": "intermediate",
+        "tags": "listings, filter, tags"
```
Comment (ming-tsai) [inline data/ideas.json:192]: - The first tag is used to match projects with an idea, so please add the book-app tag
- Remove the programming tag, due to is only an idea so the everyone could make on whatever programming language
```suggestion
        "tags": "job-listings-filter, listings, filter, tags"
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Vinitvh,
Some suggestions for the pull request.

File: data/ideas.json [hunk 2]
```diff
@@ -184,5 +184,12 @@
...
+    },
```
Comment (ming-tsai) [inline data/ideas.json:193]: Remove unnecessary comma `,`

File: data/ideas.json (hunk 1 above)
Comment (Vinitvh) [reply data/ideas.json:192]: Thank you for the suggestion. Will make necessary changes.

**Pull request closed without merging**